stdlib/public/core/Sequence.swift

```diff
@@ -583,6 +583,18 @@ extension SequenceType where Generator.Element : Equatable {
     return split(maxSplit, allowEmptySlices: allowEmptySlices,
       isSeparator: { $0 == separator })
   }
+
+  /// Return an `Array` containing the elements of `self`
+  @warn_unused_result
+  public func uniq() -> [Generator.Element] {
+    return reduce(Array<Generator.Element>()) { (arr, element) -> [Generator.Element] in
```

Comment:
This is not actually the right way to do this, because it's O(n^2). See http://airspeedvelocity.net/2015/08/03/arrays-linked-lists-and-performance/ for details. You should switch to using a temporary array and a plain old for loop, despite the ugliness.